### PR TITLE
633 same changes multiple times

### PR DESCRIPTION
**Pull request by MichalMurin**
Bug with multiple change requests in left panel fixed:

- While adding request to the database, we are checking wheter there is already same request, if yes, second request will not be added to the database -> no duplicities in the database and no multiple requests showing in the left panel of application.

### CODE REVIEW

**Comment by Rado996** [comment]
Good job

File: Backend/BlockChangesService.cs [hunk 1]
```diff
@@ -57,10 +57,31 @@ public async Task<bool> CancelExchangeRequest(BlockChangeRequest request)
             return (a != null);
         }
 
-        private async Task AddAsync(BlockChangeRequest entityToAdd)
+        private async Task<BlockChangeRequest> AddAsync(BlockChangeRequest entityToAdd)
```
Comment (marektavac) [inline Backend/BlockChangesService.cs:60]: The method should be renamed, it is not just add, may be: FindOrAdd...

**Review by marektavac — CHANGES_REQUESTED** [review]
Good idea how to fix it...
Just some small change required

File: Backend/BlockChangesService.cs (hunk 1 above)
Comment (MichalMurin) [reply Backend/BlockChangesService.cs]: Method is renamed to FindOrAdd(...)

**Review by marektavac — APPROVED** [review]

**Review by SimonAlfons71830 — APPROVED** [review]

**Pull request merged**